data/sql/updates/pending_db_world/rev_1722404655301622500.sql

```diff
@@ -0,0 +1,82 @@
+-- make sure GO exists properly
+DELETE FROM `gameobject_template` WHERE (`entry` = 185549);
+INSERT INTO `gameobject_template` (`entry`, `type`, `displayId`, `name`, `IconName`, `castBarCaption`, `unk1`, `size`, `Data0`, `Data1`, `Data2`, `Data3`, `Data4`, `Data5`, `Data6`, `Data7`, `Data8`, `Data9`, `Data10`, `Data11`, `Data12`, `Data13`, `Data14`, `Data15`, `Data16`, `Data17`, `Data18`, `Data19`, `Data20`, `Data21`, `Data22`, `Data23`, `AIName`, `ScriptName`, `VerifiedBuild`) VALUES
+(185549, 10, 378, 'Monstrous Kaliri Egg', '', '', '', 1.5, 0, 11008, 0, 3000, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', 12340);
```

Comment:
This change is incorrect. `Data5` has been changed to a 1, in a 52237 sniff, it is meant to be a 0.

Reply:
Data5 controls whether the GameObject can be despawned. if its 0 it will only play the action animation and then revert its original state.

Former implementation had trigger mobs on each spawn spot and they were responsible of spawning via timed action spell cast. When i tried to make it work via mobs i had limited success because the despawn of the GameObjects was unreliable and sometimes it spawn multiple of them at the same location. So i removed the mobs completely and used servers GameObjects and spawns instead. It worked way better and without issues so far however the flip side is that the implementation is not according to sniffs.